todoman/configuration.py

```diff
@@ -31,6 +47,8 @@ def load_config():
 
 
 def _load_config_impl(path):
-    config = ConfigParser(interpolation=None)
-    config.read(path)
-    return config
+    with open(path) as conffile:
+        config = toml.loads(conffile.read())
+    # TODO: Validate required fields here (ie: path)
```

Comment:
Huh, yeah, I'll look at validating the config file, since that *would* be a stronger bonus. But I don't see how to do this without another dependency (python-jsonschema).